src/metabase/task.clj

```diff
@@ -140,7 +139,7 @@
   connection properties ahead of time, we'll need to set these at runtime rather than Setting them in the
   `quartz.properties` file.)"
   []
-  (when (= (mdb/db-type) :postgres)
+  (when (= (mdb.connection/db-type) :postgres)
```

Comment:
This was needed to avoid cyclic dependencies. `mdb` just imports the var from `mdb.connection` for convenience anyway